watchtower-plugin/src/net/http.rs

```diff
@@ -154,6 +155,31 @@ pub async fn send_appointment(
     }
 }
 
+/// Handles a ping request to the tower.
+pub async fn ping(server_url: &str) -> Result<(), String> {
+    let client = Client::new();
```

Comment:
I think you should split this into the ping function and an underlying `get_request` in the same fashion that `post_request` works. That way we'll be able to re-use it if needed for future additions

Reply:
oh that's a great idea. Thanks!